cypress/integration/dashboard.spec.js

```diff
@@ -1,18 +1,37 @@
 /// <reference types="cypress" />
 
-describe('home page loads', () => {
+describe('A11y', () => {
+    it('dashboard page is accessible', () => {
+      cy.checkPageA11y('/dashboard')
+    })
+  });
+
+describe('The dashboard page loads', () => {
     beforeEach(() => {
       cy.visit('/dashboard')
-    //   cy.injectAxe();
+       cy.injectAxe();

```

Comment:
`checkPageA11y` is already injecting Axe I think, right?

or maybe it's unique to each `describe` block. If so, do we need both `describe` blocks? can we merge them into one?

Reply:
Normally it would be only one but I put them separate as I am ignoring the error to catch the 400 and I didn't want that always.

Reply:
Ok fair enough! If the team has thoughts on this we can come back to it!